nvtabular/ops/categorify.py

```diff
@@ -764,7 +784,8 @@ def _encode(name, storage_name, path, gdf, cat_cache, na_sentinel=-1, freq_thres
                 gdf[selection_l], side="left", na_position="first"
             )
         labels[labels >= len(value[selection_r])] = na_sentinel
-
+    if search_sorted is True and freq_threshold > 0:
+        raise ValueError("cannot use search_sorted=True with freq_threshold > 0.")
```

Comment:
We probably want this check in the constructor - I think at this stage we will have already calculated the categories and applied them to at least one partition before failing,

Reply:
@benfred thanks, yes Rick also mentioned that, so I added this condition into the constructor. will push the changes.